pkg/webservice/handlers.go

```diff
@@ -823,7 +823,7 @@ func getUsersResourceUsage(w http.ResponseWriter, _ *http.Request) {
 	writeHeaders(w)
 	userManager := ugm.GetUserManager()
 	usersResources := userManager.GetUsersResources()
-	var result []*dao.UserResourceUsageDAOInfo
+	result := []*dao.UserResourceUsageDAOInfo{}
```

Comment:
how about using `make` with initial capacity? 

```
result := make([]*dao.UserResourceUsageDAOInfo, len(usersResources))
```

Reply:
+1 for that

Reply:
Actually I think it has to be `result := make([]*dao.UserResourceUsageDAOInfo, 0, len(usersResources))` so that appending starts at position 0.

Reply:
The even faster solution is to use this:
```
	result := make([]*dao.UserResourceUsageDAOInfo, len(usersResources))
	for i, tracker := range usersResources {
		result[i] = tracker.GetUserResourceUsageDAOInfo()
	}
```

Reply:
Already addressed comments, thank you for everyone's valuable suggestions !